java/server/src/org/openqa/selenium/grid/console/public/css/fontawesome.min.css

```diff
@@ -0,0 +1,5 @@
+/*!
```

Comment:
Same for this.

Reply:
same for this.

Reply:
We want the site to work without having to fetch anything over the internet. The UI might be behind firewalls and might not have access to the internet